roles/common/templates/ubuntu.list

```diff
@@ -0,0 +1,9 @@
+# Ubuntu Main Archive
```

Comment:
Yes they are required. Source: https://docs.midonet.org/docs/v5.0/en/quick-start-guide/ubuntu-1404_liberty/content/_repository_configuration.html
I know it's not in template format, but I did not find a better place to put this since it's a common task.